lib/puppet/type/scheduled_task.rb

```diff
@@ -83,6 +83,15 @@ def insync?(current)
       to determine if a scheduled task is in sync or not."
   end
 
+  newproperty(:compatibility) do
```

Comment:
I think we should update the README in this commit to document this.

Do we want to add the relevant bit to `CHANGELOG` too?

Reply:
Adding both is a good idea.